### PR TITLE
clean up bpmn landing page

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Closes https://github.com/camunda/camunda-docs/issues/1048. Removes gif and replaces with text, updates BPMN landing page.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/components/modeler/bpmn/bpmn.md [hunk 1]
```diff
@@ -17,15 +19,15 @@ Web and Desktop Modeler both offer a similar core BPMN 2.0 Modeling experience:
 - Add BPMN elements from the palette on the left side of the page by dragging and dropping them onto the diagram canvas.
 - Change the type of element in place by clicking on an element to reveal the context menu. Then, click the wrench icon to change the type of element to a [service task](./service-tasks/service-tasks.md) or [user task](./user-tasks/user-tasks.md), for example.
 
-## Demo
+## Creating BPMN elements
 
-![Demo](./assets/demo.gif)
+Using BPMN in Modeler, you can create more BPMN 2.0 elements like task types and event definitions.
```
Comment (akeller) [inline docs/components/modeler/bpmn/bpmn.md:24]: BPMN 2.0 is mentioned on this page, but should we link to it the first time it's referenced? Or is linking to the OMG BPMN page at the top sufficient? I'm on the fence, but this isn't blocking feedback.

**Review by akeller — DISMISSED** [review]
I left a nitpicky comment, but I also expected to see GIF files removed in this PR. Are you done?

**Comment by christinaausley** [comment]
> but I also expected to see GIF files removed in this PR

@akeller You mean `Demo`, right? That gif is removed. However, see [this comment](https://github.com/camunda/camunda-docs/issues/1048#issuecomment-2273977860) for maybe some context?

File: docs/components/modeler/bpmn/bpmn.md (hunk 1 above)
Comment (christinaausley) [reply docs/components/modeler/bpmn/bpmn.md:24]: Linking on first reference at the top of this page should be sufficient -- will adjust.

**Comment by akeller** [comment]
The file is still in the branch. `modeler > bpmn > assets > demo.gif`

We should remove the file if we don't use it anywhere else.

**Comment by christinaausley** [comment]
> The file is still in the branch. `modeler > bpmn > assets > demo.gif`
> 
> We should remove the file if we don't use it anywhere else.

@akeller AH! I see. Adjusting!

**Review by volodymyr-melnykc — APPROVED** [review]

**Comment by christinaausley** [comment]
Within the original scope of the issue, this is done. Closing this out 👍

**Pull request merged**